k2/csrc/fsa_utils.cu

```diff
@@ -1237,6 +1237,17 @@ void BackpropGetArcPost(FsaVec &fsas, Ragged<int32_t> &incoming_arcs,
       });
 }
 
+template <>
```

Comment:
This is a specialization, not an instantiation.

`BackpropGetArcPost` is never compiled.

Fixed in #599.

Reply:
Yeah, noted I fixed in PR https://github.com/k2-fsa/k2/pull/600, there may be conflicts with you, sorry.